packages/example-codehub/src/codehub-application.ts

```diff
@@ -3,12 +3,42 @@
 // This file is licensed under the MIT License.
 // License text available at https://opensource.org/licenses/MIT
 
-import {Application} from '@loopback/core';
+import {Provider} from '@loopback/context';
+import {Application, DefaultSequence, FindRoute, InvokeMethod,
+  ParsedRequest, Reject, Send, inject} from '@loopback/core';
+import {Logger, PinoLoggerComponent} from '@loopback/logger';
 import {UserController, HealthController} from './controllers';
+import {ServerResponse} from 'http';
+
+const setupLoggerKey = Logger.SequenceActions.SETUP_LOGGER;
+
+class AppSequence extends DefaultSequence {
+  private defaultHandle: Function;
+  private httpLogger: any;
+  constructor(
+    @inject('sequence.actions.findRoute') protected findRoute: FindRoute,
+    @inject('sequence.actions.invokeMethod') protected invoke: InvokeMethod,
+    @inject('sequence.actions.send') public send: Send,
+    @inject('sequence.actions.reject') public reject: Reject,
+    @inject(setupLoggerKey) public setupLogger: any,
+  ) {
+    super(findRoute, invoke, send, reject);
+    this.httpLogger = setupLogger;
+    this.defaultHandle = super.handle;
+  }
+
+  async handle(req: ParsedRequest, res: ServerResponse) {
+    this.httpLogger(req, res);
+    await this.defaultHandle(req, res);
```

Comment:
> because tsc complains:
>
> - Constructors for derived classes must contain a 'super' call.
> - Super calls are not permitted outside constructors or in nested functions inside constructors.

Sorry, my mistake. I think you need to call `super.handle(req, res)`. Will that work?